test/core/src/temalab/logger/LoggerBase.java

```diff
@@ -0,0 +1,56 @@
+package temalab.logger;
+import java.io.PrintStream;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public abstract class LoggerBase implements Logger {
+    private PrintStream printStream = null;
```

Comment:
ha a `LoggerBase` nem használja a `printStream`et, akkor ne is tartalmazza, mert csak egy felesleges függőség